### PR TITLE
Update changelog for release v0.23.0

### PR DESCRIPTION
**Pull request by dsimansk**
## Changes

* Update changelog for release v0.23.0

### CODE REVIEW

**Review by navidshaikh — APPROVED** [review]
thanks David!

/lgtm

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -12,18 +12,51 @@
...
+| Drop --strict flag on ko
+| https://github.com/knative/client/pull/1295[#1295]
+
+| 🐛
+| Fix documentation of the configuration options
+| https://github.com/knative/client/pull/1297[#1297]
+
+| ✨
+| Refactor waiting loop to use resourceVersion
+| https://github.com/knative/client/pull/1301[#1301]
```
Comment (rhuss) [inline CHANGELOG.adoc:58]: Would you mind to sort the entries according to issue number, please ?

**Review by rhuss — CHANGES_REQUESTED** [review]
thanks a lot ! One minor nit pick ....

File: CHANGELOG.adoc (hunk 1 above)
Comment (dsimansk) [reply CHANGELOG.adoc]: Fixed in https://github.com/knative/client/pull/1315/commits/e2e7361f8dc46729eddb07777ba33f2cf68273d3.

**Review by rhuss — COMMENTED** [review]
Thanks ! :)

/lgtm
/approve

**Pull request merged**